Source/Relativity.Test.Helpers/Relativity.Test.Helpers/KeplerHelper.cs

```diff
@@ -151,6 +151,10 @@ private bool InstallKeplerResourceFile(string keplerDll, string fileLocation)
 					};
 
 					var results = rsapiClient.PushResourceFiles(rsapiClient.APIOptions, new List<ResourceFileRequest> { rfRequest });
+
+					//sleep for 20 seconds to allow kepler to initialize
+					System.Threading.Thread.Sleep(20000);
```

Comment:
20000 should be a constant so that other tests can use it if needed.

Reply:
Done